backend/app/categories/controller.py

```diff
@@ -68,11 +69,32 @@ async def add_item_route(category_id: CategoryEnum, item: Union[Movie, Anime, Sh
 
 
 @router.patch("/{category_id}/{item_id}", response_model=ItemInResponse, dependencies=[Depends(verify_token)], tags=["update"])
-async def update_item_route(category_id: CategoryEnum, item_id: ObjectID,
-                            db: AsyncIOMotorClient = Depends(get_database)) -> ItemInResponse:
+async def update_item_route(category_id: CategoryEnum,item_id: ObjectID, 
```

Comment:
The `item_id` is already being checked for validity be declaring it of this custom type

Reply:
I added that check as it wasn't returning an error in the response in case of invalid item_id and throwing a server error. The check for whether the item_id exists in the db doesn't catch this error. What response would be good here?

Reply:
This has been fixed in the latest PR. If you rebase it'll work, won't throw the server error